config_defaults.yml

```diff
@@ -29,13 +29,25 @@
     env: PUPPET_GEM_VERSION="~> 3.0"
     bundler_args: --without system_tests
 Gemfile:
+  supports_windows:
+    - puppetlabs-acl
+    - puppetlabs-chocolatey
+    - puppetlabs-dism
+    - puppetlabs-dsc
+    - puppetlabs-powershell
+    - puppetlabs-reboot
+    - puppetlabs-registry
+    - puppetlabs-sqlserver
+    - puppetlabs-wsus_client
```

Comment:
That's an interesting choice. I would have assumed this was going to be in a different file.